### ISSUE TITLE
False positive on BlockControlStructureSpacing

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#771]
```php
<?php

declare(strict_types=1);

while (! true) {
    echo 1;
};

do {
    echo 1;
} while (! false);

for (;;) {
    echo 'To infity and beyond';
};

```

Produces `Expected 1 lines after "while", found -1` with `<rule ref="SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing"/>` set

**Comment by kukulich** [comment]
@simPod Could you try to remove the useless semicolon after `while`?

**Comment by simPod** [comment]
@kukulich that helped. I guess there's no need to fix it then?

**Comment by kukulich** [comment]
@simPod I can problably fix it but it has very low priority :)

**Comment by simPod** [comment]
Sure, feel free to close. Thanks!

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/469368cc6d8fe83aceba259ef65f1c0a2f63055b
